### DISCUSSION TITLE
What is the difference in binaries at repo.mongodb.org vs repo.mongodb.com

### DISCUSSION THREAD

**Discussion by Noushath** [docker-library/mongo#542] [General]
Would like to understand what is the difference in binaries at repo.mongodb.org vs repo.mongodb.com, please let me know what's the difference in the binaries present in above sites are they same? Why are they hosted at 2 different sites? Is both sites updated regularly with patches?

**Comment by yosifkit** [comment]
The `.com` is for their `mongodb-enterprise-*` packages and `.org` is for the non-enterprise packages (`mongodb-org-*`). It looks like they have some packages in common, though I am unsure if they are bit-for-bit the same. There are likely differences in which versions still have updates and which OS's and architectures are supported in each (https://www.mongodb.com/docs/manual/administration/production-notes/#platform-support-matrix).

**Reply by Noushath** [reply]
When you say the binaries at `.org` are `non-enterprise` can I take that implicitly means they are `community` ones?
